### PR TITLE
Commit

### PR DESCRIPTION
**Pull request by PronovichR4**

### CODE REVIEW

**Comment by PronovichR4** [comment]
done

File: Naming/src/main/java/com/epam/engx/cleancode/naming/task1/CollectOrderService.java [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-    private NotificationManager ser2;
+    private CollectionService collectionService;
+    private NotificationManager notificationManager;
 
-    public void submitOrder(Order pOrder) {
-        if (ser1.isEligibleForCollection(pOrder))
-            ser2.notifyCustomer(Message.READY_FOR_COLLECT, 4); // 4 - info notification level
+    public void submit(Order order) {
+        if (collectionService.isEligibleForCollection(order))
+            notificationManager.notifyCustomer(Message.READY_FOR_COLLECT, 4); // 4 - info notification level
```
Comment (Yauhenda) [inline Naming/src/main/java/com/epam/engx/cleancode/naming/task1/CollectOrderService.java]: The goal of every programmer should be to write code so clean and expressive that code comments are unnecessary. 

```suggestion
            notificationManager.notifyCustomer(Message.READY_FOR_COLLECT, INFO_NOTIFICATION_LEVEL); 
```

File: Naming/src/main/java/com/epam/engx/cleancode/naming/task1/CollectOrderService.java [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
         else
-            ser2.notifyCustomer(Message.IMPOSSIBLE_TO_COLLECT, 1); // 1 - critical notification level
+            notificationManager.notifyCustomer(Message.IMPOSSIBLE_TO_COLLECT, 1); // 1 - critical notification level
```
Comment (Yauhenda) [inline Naming/src/main/java/com/epam/engx/cleancode/naming/task1/CollectOrderService.java]: Same as line 15

File: Naming/src/main/java/com/epam/engx/cleancode/naming/task5/FileExtensionsPredicate.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.epam.engx.cleancode.naming.task5;
+
+
+import com.epam.engx.cleancode.naming.task5.thirdpartyjar.Predicate;
+
+public class FileExtensionsPredicate implements Predicate<String> {
+
+    private String[] extensions;
```
Comment (Yauhenda) [inline Naming/src/main/java/com/epam/engx/cleancode/naming/task5/FileExtensionsPredicate.java]:
```suggestion
    private String[] allowedExtensions;
```

File: Naming/src/main/java/com/epam/engx/cleancode/naming/task5/thirdpartyjar/Predicate.java [hunk 4]
```diff
@@ -2,5 +2,5 @@
 
 public interface Predicate<T> {
 
-    boolean test(T fileName);
+    boolean checkExtension(T fileName);
```
Comment (Yauhenda) [inline Naming/src/main/java/com/epam/engx/cleancode/naming/task5/thirdpartyjar/Predicate.java]: Good suggestion but we can't change files in thirdpartyjar directories

**Pull request closed without merging**